mmv1/third_party/terraform/tests/resource_gke_hub_feature_membership_test.go.erb

```diff
@@ -573,6 +573,204 @@ resource "google_gke_hub_feature_membership" "feature_member" {
 `, context)
 }
 
+func TestAccGkeHubFeatureMembership_gkehubFeatureAcmOci(t *testing.T) {
+  // Multiple fine-grained resources cause VCR to fail
+  skipIfVcr(t)
+  t.Parallel()
+
+  context := map[string]interface{}{
+    "random_suffix":   randString(t, 10),
+    "org_id":          getTestOrgFromEnv(t),
+    "billing_account": getTestBillingAccountFromEnv(t),
+  }
+
+  vcrTest(t, resource.TestCase{
+    PreCheck:  func() { testAccPreCheck(t) },
+    Providers: testAccProvidersOiCS,
+    CheckDestroy: testAccCheckGKEHubFeatureDestroyProducer(t),
+    Steps: []resource.TestStep{
+      {
+        Config: testAccGkeHubFeatureMembership_gkehubFeatureAcmOciStart(context),
+        Check: resource.ComposeTestCheckFunc(
+          testAccCheckGkeHubFeatureMembershipPresent(t, fmt.Sprintf("tf-test-gkehub%s", context["random_suffix"]), "global", "configmanagement", fmt.Sprintf("tf-test1%s", context["random_suffix"])),
+        ),
+      },
+      {
+        ResourceName:      "google_gke_hub_feature_membership.feature_member",
```

Comment:
This verify is looking for `google_gke_hub_feature_membership.feature_member` which doesn't exist in the configuration you've applied above.

You likely meant `google_gke_hub_feature_membership.feature_member_1`

Reply:
Changed the TF resource ids to be inline with ResourceName.